test/e2e/streaming.go

```diff
@@ -128,7 +128,7 @@ func startLongRunningContainer(rc internalapi.RuntimeService, ic internalapi.Ima
 	imageSpec := &kubeapi.ImageSpec{
 		Image: &latestTestImageRef,
 	}
-	err = ic.PullImage(imageSpec, nil)
```

Comment:
I'm afraid the streaming url has been changed on upstream: https://github.com/feiskyer/frakti/blob/60f69a711af0c98f378fe46a7fc535dd27c79fbd/test/e2e/streaming.go#L157

See: https://github.com/kubernetes/kubernetes/pull/38742/files#diff-399a1489053b508436d8c0845b122e4eR214